debian/intel-gpgpu-dkms-ubuntu-5.14-oem.dkms.in

```diff
@@ -31,5 +31,5 @@ num_cpu_cores()
 }
 
 # Here is where we could specify use of the defconfig. Right now it uses the kernel configuration from /usr/src/
-MAKE="cp defconfigs/i915 .config; 'make' -j$(num_cpu_cores) KLIB=/lib/modules/$kernelver olddefconfig; 'make' -j$(num_cpu_cores) KLIB=/lib/modules/$kernelver"
-CLEAN="'make' clean"
+MAKE="cp defconfigs/i915 .config; 'make' -j$(num_cpu_cores) KLIB=/lib/modules/$kernelver KLIB_BUILD=$kernel_source_dir olddefconfig; 'make' -j$(num_cpu_cores) KLIB=/lib/modules/$kernelver KLIB_BUILD=$kernel_source_dir"
```

Comment:
This change is not necessary. as default is always pointing to same folder 

Makefile:KLIB_BUILD ?= $(KLIB)/build/

Reply:
The problem is it is possible that this dkms is used in some CI builder that does not install dkms packages in the standard way, so `/sbin/dkms` always specifies `$kernel_source_dir`.

Reply:
@vicamo  can you please share how to reproduce the issue and what is the setup needed to check this. what is the difference here w.r.t standard instllation vs case which this change is fixing

Reply:
The standard from `/sbin/dkms`'s point of view, is to follow the `$kernel_source_dir` variable it specified. Don't assume it will always be `$(KLIB)/build/`.